Tests/FunctionalTests/Edge+ConsentTests.swift

```diff
@@ -0,0 +1,276 @@
+//
+// Copyright 2021 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+import AEPConsent
+import AEPCore
+@testable import AEPEdge
+import AEPIdentityEdge
+import AEPServices
+import Foundation
+import XCTest
+
+class EdgeConsentTests: FunctionalTestBase {
+    private let EVENTS_COUNT: Int32 = 5
+    private let experienceEvent = ExperienceEvent(xdm: ["test": "xdm"])
+    private let responseBody = "\u{0000}{" +
+        "      \"requestId\": \"e94835b0-e0b1-4038-b943-80d98fd9e2c7\"," +
+        "      \"handle\": [" +
+        "        {" +
+        "          \"payload\": [" +
+        "            {" +
+        "              \"key\": \"kndctr_example_AdobeOrg_consent\"," +
+        "              \"value\": \"\"," +
+        "              \"maxAge\": 5552000" +
+        "            }," +
+        "            {" +
+        "              \"key\": \"kndctr_example_AdobeOrg_identity\"," +
+        "              \"value\": \"abcd\"," +
+        "              \"maxAge\": 34128000" +
+        "            }," +
+        "            {" +
+        "              \"key\": \"kndctr_example_AdobeOrg_consent_check\"," +
+        "              \"value\": \"1\"," +
+        "              \"maxAge\": 7200" +
+        "            }" +
+        "          ]," +
+        "          \"type\": \"state:store\"" +
+        "        }" +
+        "      ]" +
+        "    }\n"
+
+    public class override func setUp() {
+        super.setUp()
```

Comment:
Do we need to call `super.setUp()`? I don't recall seeing these before.

Reply:
yes, we are overriding both class & test setUp functions in FunctionalTestBase, so this will call the base class setUp first:
```
demo - class setUp FunctionalTestBase
demo - class setUp my test
demo - setUp FunctionalTestBase
demo - setUp my test
```